plugins/de.cognicrypt.core/src/de/cognicrypt/utils/Utils.java

```diff
@@ -295,7 +295,7 @@ public static boolean isSubType(String typeOne, String typeTwo) {
 				subTypes = Class.forName(typeOne).isAssignableFrom(Class.forName(typeTwo));
 			}
 			catch (ClassNotFoundException e) {
-				Activator.getDefault().logError(e);
+				// It's fine if above throws a ClassNotFoundException

```

Comment:
Why? Where is the Error handled above?

Reply:
The logic works like "If both the type strings `typeOne` and `typeTwo` are convertible to classes, then check for subtype relation. Otherwise they are assumed to be primitive or primitive arrays and existing `subTypes` value calculated previously is returned".